mteb/abstasks/AbsTask.py

```diff
@@ -87,24 +134,10 @@ def load_data(self, **kwargs):
         self.data_loaded = True
 
     @property
-    def metadata_dict(self) -> dict[str, str]:
+    def metadata_dict(self) -> dict[str, Any]:
         metadata_dict = dict(self.metadata)
         return metadata_dict
```

Comment:
```suggestion
        return dict(self.metadata)
```

Reply:
fixed in https://github.com/embeddings-benchmark/mteb/pull/770